docs/makerd.md

```
@@ -194,23 +223,26 @@ This will launch `makerd` and connect it to the Bitcoin RPC core running on its
+  INFO openswap::wallet::api - Wallet file at "/path/to/maker" successfully loaded.
   ```
 
-- **New Wallet Creation**: If no wallet file is found, `makerd` will create a new wallet named `maker-wallet`:
+- **New Wallet Creation**: If no wallet file is found, `makerd` will create a new wallet named `maker`. Wallet files are always encrypted, so a wallet passphrase must be supplied when creating (and later opening) the wallet. Prefer the `OPENSWAP_WALLET_PASSWORD` environment variable over `-p`/`--PASSWORD` — a command-line value is visible in the process list and shell history:
 
   ```bash
-  INFO openswap::wallet::api - Backup the Wallet Mnemonics.
-  ["harvest", "trust", "catalog", "degree", "oxygen", "business", "crawl", "enemy", "hamster", "music", "this", "idle"]
-  
-  INFO openswap::maker::api - New Wallet created at: "$HOME/.openswap/maker/wallets/maker-wallet".
+  $ OPENSWAP_WALLET_PASSWORD=my-wallet-passphrase ./makerd -a user:password -r 127.0.0.1:38332
```

markdownlint-cli2 (0.23.2)

[warning] Dollar signs used before commands without showing output

(MD014, commands-show-output)